test/expect-tests/dune_console/dune_console_tests.ml

```diff
@@ -95,10 +113,10 @@ let%expect_test "Status line clearing with wrapping." =
   let module Console = New () in
   Console.Backend.set Console.Backend.progress;
   test_status_line_clearing_with_wrapping (module Console);
+  escape [%expect.output];
   [%expect
-    "\n\
-     This status line is a problem because of the fact that it is especially long and \
-     therefore will not be cleared \
-     properly.\r                                                                                                                        \
-     \rHere is another\r               \r "]
+    {| 
+This status line is a problem because of the fact that it is especially long
+and therefore will not be cleared properly.\r\027[2K\027M\r\027[2K\027M\r\027[2KHere is another\r\027[2K\027M\r\027[2K
```

Comment:
@rgrinberg Do you see what I mean about adding the space after `\r` making this easier to read?

Reply:
It isn't easy enough. As I've mentioned, this check is complex enough that it can't be done visually in an effective way. You should write some code that tests the assertion you're trying to eyeball.